chatbot/src/main/java/org/egov/chat/xternal/systeminitiated/PGRStatusUpdateEventFormatter.java

```diff
@@ -318,8 +318,8 @@ private JsonNode responseForRejectedStatus(JsonNode event, String comments, Stri
     }
 
     private JsonNode responseForResolvedStatus(JsonNode event, String citizenName, String mobileNumber) throws UnsupportedEncodingException {
-        String serviceRequestId = event.at("/PGREntity/service/serviceRequestId").asText();
-        String serviceCode = event.at("/PGREntity/service/serviceCode").asText();
+        String serviceRequestId = event.at("/service/serviceRequestId").asText();
```

Comment:
Use a constant files for these path instead of repeating such paths in code for future changes